crates/ide/src/syntax_highlighting.rs

```diff
@@ -519,7 +522,11 @@ fn traverse(
                 highlight |= HlMod::Attribute
             }
             if inside_macro_call && tt_level > 0 {
-                highlight |= HlMod::Macro
+                if inside_proc_macro_call {
+                    highlight |= HlMod::ProcMacro
+                } else {
+                    highlight |= HlMod::Macro
+                }
```

Comment:
```suggestion
                if inside_proc_macro_call {
                    highlight |= HlMod::ProcMacro
                }
                highlight |= HlMod::Macro
```
I think this makes more sense, that way one doesn't have to handle the proc macro modifier specifically if they are only interested in macros in general